examples/structured_2d_dgsem/elixir_advection_coupled.jl

```diff
@@ -48,70 +49,86 @@ cells_per_dimension1 = cells_per_dimension
+                                                            y_neg = boundary_condition_periodic,
+                                                            y_pos = boundary_condition_periodic))
 
 # Second mesh is the right half of a [-1,1]^2 square
 coordinates_min2 = (0.0, -1.0) # minimum coordinates (min(x), min(y))
-coordinates_max2 = (1.0,  1.0) # maximum coordinates (max(x), max(y))
+coordinates_max2 = (1.0, 1.0) # maximum coordinates (max(x), max(y))
 
 cells_per_dimension2 = cells_per_dimension
 
 mesh2 = StructuredMesh(cells_per_dimension2, coordinates_min2, coordinates_max2)
 
-semi2 = SemidiscretizationHyperbolic(mesh2, equations, initial_condition_convergence_test, solver,
-                                     boundary_conditions=(
-                                       # Connect left boundary with right boundary of left mesh
-                                       x_neg=BoundaryConditionCoupled(1, (:end, :i_forward), Float64),
-                                       # Connect right boundary with left boundary of left mesh
-                                       x_pos=BoundaryConditionCoupled(1, (:begin, :i_forward),  Float64),
-                                       y_neg=boundary_condition_periodic,
-                                       y_pos=boundary_condition_periodic))
+coupling_function2 = coupling_converter_identity(equations)
+
+boundary_conditions_x_neg2 = BoundaryConditionCoupled(1, (:end, :i_forward), Float64,
+                                                      coupling_function2)
+boundary_conditions_x_pos2 = BoundaryConditionCoupled(1, (:begin, :i_forward), Float64,
+                                                      coupling_function2)
+
+semi2 = SemidiscretizationHyperbolic(mesh2, equations, initial_condition_convergence_test,
+                                     solver,
+                                     boundary_conditions = (
+                                                            # Connect left boundary with right boundary of left mesh
+                                                            x_neg = boundary_conditions_x_neg2,
+                                                            # Connect right boundary with left boundary of left mesh
+                                                            x_pos = boundary_conditions_x_pos2,
+                                                            y_neg = boundary_condition_periodic,
+                                                            y_pos = boundary_condition_periodic))
 
 # Create a semidiscretization that bundles semi1 and semi2
 semi = SemidiscretizationCoupled(semi1, semi2)
 
 ###############################################################################
 # ODE solvers, callbacks etc.
 
-# Create ODE problem with time span from 0.0 to 2.0
-ode = semidiscretize(semi, (0.0, 2.0));
+# Create ODE problem with time span from 0.0 to 20.0
+ode = semidiscretize(semi, (0.0, 20.0));
```

Comment:
Is it really useful to have the default time to be so long or would it suffice to use a shorter time here and just let it run longer where desired?